starknet_py/net/models/transaction.py

```diff
@@ -72,6 +80,45 @@ class AccountTransaction(Transaction, ABC):
 TypeAccountTransaction = TypeVar("TypeAccountTransaction", bound=AccountTransaction)
 
 
+@dataclass(frozen=True)
+class DeclareV2(AccountTransaction):
+    """
+    Represents a transaction in the Starknet network that is a declaration of a Starknet contract
+    class.
```

Comment:
```suggestion
    Represents a transaction in the Starknet network that is a version 2 declaration of a Starknet contract
    class.
```
or something among the lines